### PR TITLE
Add support for the CONNREFUSED socket error

### PR DESCRIPTION
**Pull request by LotP1**

### CODE REVIEW

File: src/core/hle/service/sockets/sockets.h [hunk 1]
```diff
@@ -19,6 +19,7 @@ enum class Errno : u32 {
     INVAL = 22,
     MFILE = 24,
     PIPE = 32,
+    CONNREFUSED = 61,
```
Comment (liamwhite) [inline src/core/hle/service/sockets/sockets.h:22]:
```suggestion
    CONNREFUSED = 111,
```
Where did you get 61?

**Review by liamwhite — CHANGES_REQUESTED** [review]

File: src/core/hle/service/sockets/sockets.h (hunk 1 above)
Comment (LotP1) [reply src/core/hle/service/sockets/sockets.h:22]: Interesting, i got 61 from the FreeBSD manual, but looking around i see most other places using 111 for CONNREFUSED.

Comment (liamwhite) [reply src/core/hle/service/sockets/sockets.h:22]: The errno constants used in the Switch come from Android and are therefore Linux constants.

**Comment by LotP1** [comment]
> Just for the record, did you find this error code being used anywhere?

yeah, I'm writing a mod that's using sockets, and this issue is causing my Connect to return success when it's fails with CONNREFUSED.
i got the error code from the assert on line 39 in sockets_translate.cpp

**Review by liamwhite — APPROVED** [review]

**Pull request merged**